open-craft/opencraft
# BB-2240: Custom page editor

This PR adds the Custom Page editor functionality in the new Ocim UX.

**Testing Instructions:**
1. Check out this branch.
2. Rebuild API client.
3. Run Ocim and new frontend.
4. Go to the custom pages and test around.

Some edge cases for testing:
* Change the page content, them click on a link for another page. 
* Go to some page that has some text (http://localhost:3000/console/custom-pages/honor), do a browser refresh and then try to save the content without changing.
* Do the same as above, but change the content.
* Try erasing values.

**Reviewer:**
